Flaky Test: chainerx_tests.unit_tests.routines_tests.test_trigonometric_hyperbolic.test_Arcsin

[Jenkins](https://jenkins.preferred.jp/job/chainer/job/chainer_pr/1538/TEST=CHAINERX_chainerx-py3,label=mn1-p100/console)

tests/chainerx_tests/unit_tests/routines_tests/test_trigonometric_hyperbolic.py

https://jenkins.preferred.jp/job/chainer/job/chainer_pr/2114/TEST=CHAINERX_chainerx-py3,label=mn1-p100/console

`19:15:28 FAIL ../../../repo/tests/chainerx_tests/unit_tests/routines_tests/test_trigonometric_hyperbolic.py::test_Arcsin_param_185_{contiguous='C', in_dtypes=('float32',), input=-0.9, out_dtype='float32', shape=(2, 3)}[native:0]`


```
19:15:27 E           chainer.testing.function_link.FunctionTestError: backward is not implemented correctly
19:15:27 E           
19:15:27 E           (caused by)
19:15:27 E           AssertionError: check_backward failed (eps=0.001 atol=1e-05 rtol=0.0001)
19:15:27 E           inputs[0]:
19:15:27 E           array([[-0.89999998, -0.89999998, -0.89999998],
19:15:27 E                  [-0.89999998, -0.89999998, -0.89999998]], shape=(2, 3), dtype=float32, device='native:0')
19:15:27 E           grad_outputs[0]:
19:15:27 E           array([[0.75203574, -0.83166498, -0.21992885],
19:15:27 E                  [-0.26165313, 0.87954009, -0.87090617]], shape=(2, 3), dtype=float32, device='native:0')
19:15:27 E           directions[0]:
19:15:27 E           array([[0.45869688, -0.12954883, -0.13890619],
19:15:27 E                  [-0.02411799, -0.82449974, -0.27044009]], shape=(2, 3), dtype=float64, device='native:0')
19:15:27 E           gradients (numeric):  array(-0.0002289, shape=(), dtype=float64, device='native:0')
19:15:27 E           gradients (backward): array(-0.0002177, shape=(), dtype=float64, device='native:0')
19:15:27 E           
19:15:27 E           x: numeric gradient, y: backward gradient
19:15:27 E           Not equal to tolerance rtol=0.0001, atol=1e-05
19:15:27 E           
19:15:27 E           Mismatch: 100%
19:15:27 E           Max absolute difference: 1.11929556e-05
19:15:27 E           Max relative difference: 0.05141286
19:15:27 E            x: array(-0.000229)
19:15:27 E            y: array(-0.000218)
19:15:27 E           
19:15:27 E           assert_allclose failed: 
19:15:27 E             shape: () ()
19:15:27 E             dtype: float64 float64
19:15:27 E             i: (0,)
19:15:27 E             x[i]: -0.00022890024923487122
19:15:27 E             y[i]: -0.00021770729362868835
19:15:27 E             relative error[i]: 0.051412864583549815
19:15:27 E             absolute error[i]: 1.1192955606182872e-05
19:15:27 E             relative tolerance * |y[i]|: 2.1770729362868835e-08
19:15:27 E             absolute tolerance: 1e-05
19:15:27 E             total tolerance: 1.002177072936287e-05
19:15:27 E           x: -0.0002289
19:15:27 E           y: -0.00021771
```

--- a/tests/chainerx_tests/math_utils.py
+++ b/tests/chainerx_tests/math_utils.py
@@ -34,6 +34,10 @@ class UnaryMathTestBase(object):
             self.check_backward_options.update({'rtol': 3e-3, 'atol': 3e-3})
             self.check_double_backward_options.update(
                 {'rtol': 1e-2, 'atol': 1e-2})
+        else:
+            self.check_backward_options.update({'rtol': 1e-3, 'atol': 1e-4})
+            self.check_double_backward_options.update(
+                {'rtol': 1e-3, 'atol': 1e-4})
 
         input = self.input
         if (in_kind == 'u'

